src/Lumi/Components.purs

```diff
@@ -40,8 +40,14 @@ type PropsModifier props
     (LumiProps props -> LumiProps props)
 
 -- | Lift a `props -> props` function for composition with other `PropsModifier` functions.
+-- |
+-- | NOTE: The composition of `PropsModifier`s should be read like that of an
```

Comment:
Does the "NOTE:" add anything here?

Reply:
I'd say it helps draw some attention. Maybe I should change it to `WARNING` instead?

Reply:
I dunno, probably not worth worrying about.